staff/daniel-villegas/app-template/about.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Petanca news</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="./home.html">Home</a>
+            <a href="./login.html">Login</a>
+            <a href="./news.html">News</a>
+            <a href="./register.html">Register</a>
+            <a href="./about.html">About</a>
+            <a href="./contact.html">Contact</a>
+            <a href="./profile.html">Profile</a>
```

Comment:
the anchors inside a ```<nav>``` menu are better inside a ```<li>``` inside an unordered list (```<ul>```)

Reply:
Indeed. Changed!